source/patterns/@aws-solutions-constructs/core/lib/websocket-api-helper.ts

```diff
@@ -175,8 +174,9 @@ export function buildWebSocketApiProps(
   // Sonar exception reason: - typescript:S6571 - required because we are not passing all values. Using partial may cause @jsii to not work.
   const websocketApiProps: apigwv2.WebSocketApiProps = { // NOSONAR
     defaultRouteOptions: createDefaultRoute ? buildWebSocketQueueRouteOptions(role!, sqsQueue!, requestTemplate) : undefined,
+    connectRouteOptions: (defaultIamAuthorization === undefined || defaultIamAuthorization === true) ? connectRouteOptions : undefined
   };
-
+  printWarning("defaultIamAuthorization is set to false. This construct will not add any authorizers. If this is not the case, please pass the `connectRouteOptions` with an authorizer or call `addRoute` on the websocket endpoint and provide the `$connect` configuration with an authorizer");
```

Comment:
I don't see the condition that checks whether to print this or not, looks like it is printed always?

And how is this passing eslint with a line almost 300 characters long?